python/ml4ir/applications/classification/tests/test_base.py

```diff
@@ -30,21 +30,22 @@ class ClassificationTestBase(unittest.TestCase):
     Setting default arguments and context for tests .../classification/tests folder.
     """
 
-    def setUp(
-        self,
+    @classmethod
+    def setUpClass(
```

Comment:
This should further speed up our tests. We won't need to split them up to run faster.